src/contract.rs

```diff
@@ -890,32 +1014,27 @@ mod tests {
     #[test]
     fn is_marker_admin_returns_false_without_admin_permission() {
         let non_admin_address = Addr::unchecked("some_address_without_admin");
-        let marker_json = b"{
-              \"address\": \"tp1l330sxue4suxz9dhc40e2pns0ymrytf8uz4squ\",
-              \"coins\": [
-                {
-                  \"denom\": \"restricted_1\",
-                  \"amount\": \"1000\"
-                }
-              ],
-              \"account_number\": 10,
-              \"sequence\": 0,
-              \"permissions\": [
-                {
-                  \"permissions\": [
-                    \"transfer\"
-                  ],
-                  \"address\": \"some_address_without_admin\"
-                }
-              ],
-              \"status\": \"active\",
-              \"denom\": \"restricted_1\",
-              \"total_supply\": \"1000\",
-              \"marker_type\": \"restricted\",
-              \"supply_fixed\": false
-            }";
-
-        let test_marker: Marker = from_binary(&Binary::from(marker_json)).unwrap();
+        let test_marker: MarkerAccount = MarkerAccount {
+            base_account: Some(BaseAccount {
+                address: "tp1l330sxue4suxz9dhc40e2pns0ymrytf8uz4squ".to_string(),
+                pub_key: None,
+                account_number: 10,
+                sequence: 0,
+            }),
+            manager: "tp13pnzut8zdjaqht7aqe7kk4ww5zfq04jzlytnmu".to_string(),
+            access_control: vec![AccessGrant {
+                address: "some_address_without_admin".to_string(),
+                permissions: vec![(Access::Transfer) as i32],
+            }],
+            status: 0,
```

Comment:
probably doesn't matter for the test, but the prior value was `active` instead of `unspecificed` - https://github.com/provenance-io/provenance/blob/main/docs/proto-docs.md#markerstatus